client/www/components/dash/sandbox/Data.tsx

```diff
@@ -0,0 +1,31 @@
+import Json from '@uiw/react-json-view';
+
+export function Data({
+  data,
+  collapsed,
+}: {
+  data: any;
+  collapsed?: boolean | number;
```

Comment:
nit: should we make this just 'boolean'? (Not sure why we would want a number here)